Source/Fuse.Controls.Native/DarkMode.uno

```diff
@@ -66,20 +70,59 @@ namespace Fuse.Controls.Native.iOS
 		{
 			if (_instance != null) return;
 			Resource.SetGlobalKey(_instance = this, "FuseJS/DarkMode");
+
+			SetupAndroidListener();
 		}
 
-		/* 
-			Example call for when implementing Android version:
-			@{Fuse.Controls.Native.iOS.DarkMode.changeDarkMode(string):Call(@"Dark")};
-		*/
+		[Foreign(Language.Java)]
+		public extern(Android) void SetupAndroidListener() 
+		@{ 
+			@{checkForDarkThemeChange():Call()};
+
+			com.fuse.Activity.SubscribeToLifecycleChange(new com.fuse.Activity.ActivityListener()
+			{
+				@Override public void onStop() {}
+				@Override public void onStart() {}
+				@Override public void onWindowFocusChanged(boolean hasFocus) {}
+				@Override public void onPause() {}
+				@Override public void onResume() {}
+				@Override public void onDestroy() {}
+
+				@Override public void onConfigurationChanged(android.content.res.Configuration config) 
+				{ 
+					@{checkForDarkThemeChange():Call()};
+				}
+			});
+		@}
+		
+		public extern(!MOBILE) void SetupAndroidListener() @{ debug_log("SetupAndroidListener not supported on this platform."); @}
+
+		public extern(iOS) void SetupAndroidListener() @{ @}
+
+		[Foreign(Language.Java)]
+		public static extern(Android) void checkForDarkThemeChange()
+		@{
+			switch (com.fuse.Activity.getRootActivity().getResources().getConfiguration().uiMode & android.content.res.Configuration.UI_MODE_NIGHT_MASK) {
+				case android.content.res.Configuration.UI_MODE_NIGHT_YES:
+					@{changeDarkMode(string):Call("Dark")};
+					break;
+				case android.content.res.Configuration.UI_MODE_NIGHT_NO:
+					@{changeDarkMode(string):Call("Light")};
+					break; 
+				case android.content.res.Configuration.UI_MODE_NIGHT_UNDEFINED:
+					@{changeDarkMode(string):Call("Unspecified")};
+					break; 
+			}
+
+		@}
+
+
 		public static void changeDarkMode(string modeValue) {
```

Comment:
This is from other PR, but maybe change to `internal` instead of `public`.

Reply:
aight